### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by rucha-0307** [github/india#579] [Students]
How to claim student developer pack on github?

**Comment by iamabhaytiwari** [comment]
simply just type this URL or click on it https://education.github.com/pack after that tap on sign up for student dev pack..........fill all the details matching to your institute's data after that apply for it after a few weeks you will get the confirmation email for GitHub
![Screenshot from 2021-11-21 10-25-08](https://user-images.githubusercontent.com/85944357/142750153-b27f1eab-038c-4f1f-bb56-5241d988be7a.png) i hope it will help you to get
